docker-compose.yml

```
@@ -1,19 +1,24 @@
 version: '3'
 services:
   eddi:
-    image: labsai/eddi:latest
+    image: labsai/eddi:4.8.0-kkorsakov
```

this needs to be reverted back to 'latest'
```suggestion
    image: labsai/eddi:latest
```

---

> this needs to be reverted back to 'latest'

Reverted, thanks.